ScoreSaber/ScoreSaber.csproj

```diff
@@ -34,6 +34,14 @@
     <DisableZipRelease>True</DisableZipRelease>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="UI\Elements\Leaderboard\ProfilePictureView.cs" />
```

Comment:
Not quite sure what happened here, is this an artifact from your other PRs?

Reply:
This had existed from my original attempt at profile pictures, which i had done stupidly on main back then, this can be removed without issue